package-lock.json

```diff
@@ -19107,7 +19143,7 @@
     },
     "typescript": {
       "version": "2.8.3",
-      "resolved": "https://registry.npmjs.org/typescript/-/typescript-2.8.3.tgz",
+      "resolved": "https://yammer.pkgs.visualstudio.com/_packaging/yammer-clients/npm/registry/typescript/-/typescript-2.8.3.tgz",
```

Comment:
I'm assuming this is from recent changes to my .npmrc

@unindented, @swese44 have you seen this before?

I'm guessing we always want `registry.npmjs.org`?

Reply:
Some combination of adding `registry=https://registry.npmjs.com/` to the end of my .npmrc and running `rm -rf node_modules`, `npm cache clean --force` seems to have fixed this now.